pkg/cmd/controller/route/apiserver_authenticator.go

```diff
@@ -0,0 +1,52 @@
+package route
```

Comment:
to achieve removal of openshift-controller-manager imports I had to duplicate a lot of files (mainly for the server and genericinformers)

- Is this necessary? I think we could split repo even with files that are used by both controllers.
- since it is a duplicate copy the history is preserved, but there seems to be ways on how to achive this (https://stackoverflow.com/questions/16937359/git-copy-file-preserving-history). Please let me know if we should go this way
- also most of these files will be deleted anyway after the final refactoring

Reply:
https://github.com/openshift/openshift-controller-manager/pull/234#discussion_r940186953 has shown we do not need the genericinformers so this concerns only the server code.

since we will be deleting the server code later, I am for not preserving the history of the server files and doing just plain copy.